File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Operations/Indexes/EnableIndex.cs
using Raven.Client.Documents;
using Raven.Client.Documents.Indexes;
using Raven.Client.Documents.Operations.Indexes;

namespace Raven.Documentation.Samples.ClientApi.Operations.Indexes
{
	public class EnableIndex
	{
		private interface IFoo
		{
            /*
            #region enable_1
            public EnableIndexOperation(string indexName)
            #endregion
            */
        }

        public EnableIndex()
		{
			using (var store = new DocumentStore())
			{
                #region enable_2
                store.Maintenance.Send(new EnableIndexOperation("Orders/Totals"));
                #endregion
			}
		}
	}
}

File: Documentation/5.4/Samples/java/src/test/java/net/ravendb/ClientApi/Operations/Maintenance/Indexes/EnableIndex.java
package net.ravendb.ClientApi.Operations.Indexes;

import net.ravendb.client.documents.DocumentStore;
import net.ravendb.client.documents.IDocumentStore;
import net.ravendb.client.documents.operations.indexes.EnableIndexOperation;

public class EnableIndex {

    private interface IFoo {
        /*
        //region enable_1
        public EnableIndexOperation(String indexName)
        //endregion
        */
    }

    public EnableIndex() {
        try (IDocumentStore store = new DocumentStore()) {
            //region enable_2
            store.maintenance().send(new EnableIndexOperation("Orders/Totals"));
            //endregion
        }
    }
}
